Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor.cs

```diff
@@ -10,14 +10,26 @@ namespace Blazorise.DataGrid;
 
 partial class _DataGridPagination<TItem> : BaseComponent, IDisposable
 {
+    #region Members 
+
+    private bool dropdownColumnChooserVisible;
+
+    #endregion
+
     #region Methods
 
     private bool ShowButtonRow()
         => ButtonRowTemplate is not null && ParentDataGrid.IsButtonRowVisible;
 
+    private bool ShowColumnChooser()

```

Comment:
I think it makes more sense for this to be a getter property instead of a method. Agree?

Reply:
I personally don't mind either way. It's just not consistent with what already exists.
If I change this one to be getter, I would change ShowButtonRow also?